<ui version="4.0">
 <class>TestResultsWindow</class>
 <widget class="QDialog" name="TestResultsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Результаты теста</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #4e54c8, stop:1 #8f94fb);
    }
    QLabel {
        font-size: 14pt;
        color: white;
        margin: 10px;
    }
    QTableWidget {
        background-color: rgba(255, 255, 255, 0.8);
        border: 2px solid #3498db;
        border-radius: 10px;
        padding: 10px;
        font-size: 12pt;
        color: #2c3e50;
    }
    QHeaderView::section {
        background-color: #3498db;
        color: white;
        font-size: 12pt;
        padding: 5px;
    }
    QPushButton {
        background-color: #3498db;
        color: white;
        font-size: 12pt;
        font-weight: bold;
        border: none;
        border-radius: 5px;
        padding: 10px;
        min-width: 100px;
        margin: 10px;
    }
    QPushButton:hover {
        background-color: #2980b9;
    }
    QPushButton:pressed {
        background-color: #1abc9c;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <!-- Название теста -->
   <item>
    <widget class="QLabel" name="testTitleLabel">
     <property name="text">
      <string>Название теста</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <!-- Всего вопросов -->
   <item>
    <widget class="QLabel" name="totalQuestionsLabel">
     <property name="text">
      <string>Всего вопросов: 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <!-- Правильные ответы -->
   <item>
    <widget class="QLabel" name="correctAnswersLabel">
     <property name="text">
      <string>Правильных ответов: 0 / 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <!-- Процент правильных ответов -->
   <item>
    <widget class="QLabel" name="percentageLabel">
     <property name="text">
      <string>Процент правильных ответов: 0%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <!-- Таблица статистики -->
   <item>
    <widget class="QTableWidget" name="statisticsTable">
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Вопрос</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Правильные ответы</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Выбранные ответы</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус</string>
      </property>
     </column>
    </widget>
   </item>
   <!-- Кнопка "Завершить" -->
   <item>
    <widget class="QPushButton" name="finishButton">
     <property name="text">
      <string>Завершить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
